content/docs/handling-events.md

```diff
@@ -8,32 +8,32 @@ redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+React के एलिमेंट्स के साथ इवेंट्स हैंडल करना DOM के एलिमेंट्स को हैंडल करने जैसा ही है | इनमे खली कुछ सिंटैक्टिक अंतर हैं |
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* React के इवेंट्स का नाम लोअरकेस कि जगह camelCase में रक्खा जाता है |
+* JSX के साथ आप एक function को एक string कि जगह इवेंट हैंडलर कि तरह उसमें पास करते हैं |
```

Comment:
This sounds a bit odd, can you relook into this translation

Reply:
`हैं,  न कि एक` => `हैं, न कि एक`

remove extra space

Reply:
This has been resolved.